public/index.html

```diff
@@ -43,6 +43,7 @@
             </div>
         </div>
     </header>
+    <div id="error-message"></div>
```

Comment:
The error message appears in a band across the window directly below the navigation menu.